x-pack/test_serverless/functional/test_suites/common/platform_security/navigation/management_nav_cards.ts

```diff
@@ -102,7 +107,10 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
 
       describe('Organization members', function () {
-        this.tags('skipSvlOblt'); // Observability will not support custom roles
+        // Observability will not support custom roles
+        // Cannot test cloud link on MKI (will redirect to login)
+        this.tags(['skipSvlOblt', 'skipMKI']);
+
```

Comment:
Can't mark the line directly as it's a bit further down this file, so commenting here:
Did you plan to also remove the `// `--xpack.cloud.organization_url: '/account/members'`,` comment in this sub-suite similar to how you removed it above?